command/talk.go

```diff
@@ -62,6 +62,7 @@ func speakCmd(ctx context.Context, robo *Robot, call *Invocation, effect string)
 	// block the generated message from being later recognized as a meme.
 	call.Channel.Memery.Block(call.Message.Time(), s)
 	robo.Log.InfoContext(ctx, "speak", "in", call.Channel.Name, "text", m, "emote", e)
+	robo.Metrics.SpeakLatency.Observe(time.Since(start).Seconds(), call.Channel.Name)
```

Comment:
`call.Channel.Send` instead of `call.Channel.Name`. The observation will vary with the cardinality of the data set, and the data set is determined by the send tag, not the channel. (Multiple channels can share a send tag.)